src/components/molecules/Slider/style.scss

```diff
@@ -0,0 +1,69 @@
+.slider {
+    width: 100%;
+    height: auto;
```

Comment:
`height` default value is already `auto`, so stating `height: auto` doesn't give any new value to your styles

Reply:
Removed height: auto;